pkgs/desktops/gnome/apps/gnome-logs/default.nix

```diff
@@ -41,10 +41,10 @@ stdenv.mkDerivation rec {
     libxslt
     docbook_xsl
     docbook_xml_dtd_43
+    glib
   ];
 
   buildInputs = [
-    glib
```

Comment:
Should be here as well, do not rely on propagation: https://gitlab.gnome.org/GNOME/gnome-logs/-/blob/e0a8ab855f4ea318f2d2c5dfb1b314ac2702c987/meson.build#L64